include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -949,12 +1064,51 @@ Parameters::print_parameters_turbulence(dealii::ConditionalOStream const & pcout
 {
   pcout << std::endl << "Turbulence:" << std::endl;
 
-  print_parameter(pcout, "Use turbulence model", use_turbulence_model);
+  print_parameter(pcout, "Use turbulence model", viscosity_model_data.use_turbulence_model);
```

Comment:
I think it is time to introduce `print()` functions to viscosity model data, potentially hierarchically with print functiosn for turbulence model, generalized newtonian.